packages/clay-dropdown/src/items_validator.js

```diff
@@ -3,6 +3,7 @@ import {Config} from 'metal-state';
 let itemShape = {
 	active: Config.bool().value(false),
 	checked: Config.bool().value(false),
+	data: Config.object(),
```

Comment:
Hey @carloslancha, This `data` is correspond to `dataAttributes`?

Reply:
I added wrong files in the SF commit...